graphql-dgs-platform/build.gradle.kts

```diff
@@ -47,14 +47,13 @@ dependencies {
     constraints {
         // GraphQL Platform
         api("com.graphql-java:graphql-java") {
-            version { require("18.1") }
+            version { require("19.2") }
```

Comment:
Resolves a known and common dependency conflict.

> GraphQL Java now shades Antlr runtime to prevent any further dependency conflicts. Antlr is used internally for parsing and validating of GraphQL requests and SDL. https://github.com/graphql-java/graphql-java/pull/2854

Reply:
We should upgrade 19.2 in a separate PR that can be merged before this, wtdy @srinivasankavitha